XavierGuertin/WildfireVisualizationProject
# #101 - feat(frontend): Added Load Dataset Button in MapMetaData

### Summary
Modified AvailableDatasets _(removed progress bar)_, MapMetaData _(added props and button)_ and Layout _(added separate handles, one for loading mapmetadata and one for load dataset)_.

### Changes Made
- Load Dataset button in MapMetaData
- Loading bar only appears after clicking Load Dataset button

### Testing Instructions
1. Run the application and navigate to the available datasets section
2. Click on a dataset
3. Click on Load Dataset button
4. Verify that the loading overlay appears
5. Confirm that the loading overlay disappears once the dataset is fully loaded or after a set period

### Screenshots
![image](https://github.com/user-attachments/assets/b081787b-ec88-447a-bf18-c486a56d6467)

### Additional Notes
Only frontend, added comments for foundation to backend implementation
